xhr/resources/upload.py

```diff
@@ -1,15 +1,17 @@
+from wptserve.utils import isomorphic_decode
+
 def main(request, response):
     content = []
 
-    for key, values in sorted(item for item in request.POST.items() if not hasattr(item[1][0], "filename")):
-        content.append("%s=%s," % (key, values[0]))
-    content.append("\n")
+    for key, values in sorted(item for item in request.POST.items() if not hasattr(item[1][0], u"filename")):
+        content.append(u"%s=%s," % (isomorphic_decode(key), str(values[0])))
+    content.append(u"\n")
```

Comment:
Would it be easier/cleaner to make `content` a bytes string?

Reply:
Code updated. Thank you!